std/hash/auto_hash.zig

```diff
@@ -355,6 +355,8 @@ test "testHash union" {
 }
 
 test "testHash vector" {
+    if (@import("builtin").arch == .mipsel) return error.SkipZigTest;
```

Comment:
Would you mind opening a placeholder issue for these disabled tests, and link to the issue here and in all the other places tests are disabled? e.g.  #3288 but for mips.